src/components/Item/Item.component.jsx

```diff
@@ -1,22 +1,45 @@
-import React, { useEffect } from 'react';
-import * as firebase from '../../lib/firebase';
+// React Imports
+import React, { useState, useEffect } from 'react';
+
+// Custom Imports
 import calculateEstimate from '../../lib/estimates';
-import Checkbox from '@material-ui/core/Checkbox';
+import * as firebase from '../../lib/firebase';
 import './Item.style.scss';
 
+// Material-UI Imports
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Checkbox from '@material-ui/core/Checkbox';
+
 const Item = props => {
-  const itemName = props.name;
   const itemId = props.id.toString();
+  const secondsInDay = 86400;
+  let lastPurchasedTimeInSeconds = props.purchasedTimeInSeconds;
+  const itemName = props.name;
   const localToken = props.localToken;
   const over24 = props.over24;
   const resupplyPeriod = props.resupplyPeriod;
-  const secondsInDay = 86400;
   let lastEstimate = props.lastEstimate;
   let latestInterval = props.latestInterval;
   let numberOfPurchases = props.numberOfPurchases;
   let nextPurchaseInterval = props.nextPurchaseInterval;
   let lastPurchaseDate = props.date;
-  let lastPurchasedTimeInSeconds = 0;
+
+  // Future implementation;  currently breaks but that's a low-priority issue.
+  // const { itemName, localToken, over24, resupplyPeriod } = props;
+  // let {
+  //   lastEstimate,
+  //   latestInterval,
+  //   numberOfPurchases,
+  //   nextPurchaseInterval,
+  //   lastPurchaseDate,
+  // } = props;
```

Comment:
This most likely breaks because you're trying to destructure props into 2 differently structured objects. If all of those variables are keys within props, then you'll need to unpack it into one object. What you can do is unpack everything from props, and set the ones you want to be changeable to have local state.